### DISCUSSION TITLE
Question about tutorial (Fields)

### DISCUSSION THREAD

**Discussion by diamond-fish** [fastapi/fastapi#9940] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Annotated
from fastapi import FastAPI, Query
from pydantic import BaseModel, Field

class Item(BaseModel):
  name: str
  price: float = Field(gt=0)

@app.post('/items/')
def create_item(item: Item):
  return item
```

### Description

I noticed in [this tutorial ](https://fastapi.tiangolo.com/tutorial/body-fields/) they recommended using Field to add validation like in the sample code above.

However, I noticed this also worked:

```python
class Item(BaseModel):
    name: str
    price: Annotated[float, Query(gt=0)]
```

Does using Field have any benefits over using Annotated? Since I'm already importing Annotated, I'd prefer to use Annotated instead of adding a new import if there are no advantages to using Field.

### Operating System

Windows

### FastAPI Version

0.100.0

### Python Version

3.11.4

**Comment by YuriiMotov** [comment, marked as answer]
`price: Annotated[float, Field(gt=0)]` is new syntax you can use instead of `price: float = Field(gt=0)`.

Although it works, you shouldn't use `Query` to annotate Body model's parameters
